### PR TITLE
UF-6576: Changed logic for sending letters and removed snailmail endp…

### PR DESCRIPTION
**Pull request by MickeWallner**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     </parent>
     <groupId>se.sundsvall.messaging</groupId>
     <artifactId>api-service-messaging</artifactId>
-    <version>3.3</version>
+    <version>4.0</version>
```
Comment (foobar-1337) [inline pom.xml:13]: 4.1

File: src/integration-test/resources/LetterIT/__files/test2_ErrorFromDigital_SuccessfulSnailMail/mappings/snailmail-sender-batch.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    "url": "/snailmail-sender/send/snailmail"
+    "urlPathPattern" : "/snailmail-sender/send/batch/.*",
+    "method": "POST"
   },
   "response": {
     "headers": {
       "Content-Type": "application/json"
     },
-    "status": 200
+    "status": 201
```
Comment (foobar-1337) [inline src/integration-test/resources/LetterIT/__files/test2_ErrorFromDigital_SuccessfulSnailMail/mappings/snailmail-sender-batch.json:10]: According to API (also in this PR) HTTP 200 is the status code for successful calls.

File: pom.xml (hunk 1 above)
Comment (MickeWallner) [reply pom.xml:13]: You meant dept44 version? If so it's fixed.

Comment (foobar-1337) [reply pom.xml:13]: Yes! I meant dept44-version :-) 

Everything OK then...

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**